fe/fe-core/src/main/java/org/apache/doris/analysis/DropCatalogRecycleBinStmt.java

```diff
@@ -0,0 +1,83 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.UserException;
+
+/**
+ * Statement for drop a db/table/partition in catalog recycle bin.
+ */
+public class DropCatalogRecycleBinStmt extends DdlStmt {
```

Comment:
if u do not want impl this statement in Legacy planner. i think we could do all thing in `DropCatalogRecycleBinCommand#run` directly. it is not necessary to tran it into `DropCatalogRecycleBinStmt `.